### PR TITLE
Added different health's for each enemy type

### LINKED ISSUE #159: enemies have certain health values which are passed in `enemy_info`
i think each enemy should have a certain health value and the laser should always do `1` damage

### PR DESCRIPTION
**Pull request by jcr7467**
Now the starting health for each enemy type is defined in constants.py. The damage will be subtracted from this initial health

Also fixed bug that only allowed playing 1 game of multiplayer at a time, since room id and is_host weren't being reset at the end of a game

close #159

### CODE REVIEW

File: client/game.py [hunk 1]
```diff
@@ -518,6 +520,9 @@ def _game_over(self):
                 DARK_BLUE,
                 LIGHT_BLUE):
             self.game_state = 'start'
+            #reset room info
+            self.room_id = None
+            self.is_host = None
```
Comment (randallwc) [inline client/game.py:525]: this should be in the `def _clear_variables(self):` function

File: client/game.py [hunk 2]
```diff
@@ -307,14 +307,16 @@ def _spawn_enemies(self):
...
             # add to enemy id count
             self.enemy_id_count += 1
+            new_enemy_type = choice(self.enemy_types)
             new_enemy = Enemy(
                 randrange(0, SCREEN_WIDTH, 1),
                 100,
                 randrange(0, self.num_z_levels, 1),
                 self.num_z_levels,
-                choice(self.enemy_types),
+                new_enemy_type,
```
Comment (randallwc) [inline client/game.py:316]: why was this part changed

**Review by randallwc — CHANGES_REQUESTED** [review]

File: client/game.py (hunk 1 above)
Comment (randallwc) [reply client/game.py:525]: https://github.com/ECE-180D-WS-2022/Team3/blob/cca63b671d53befb2681e9a6775ec33b39aad017/client/game.py#L199

File: client/game.py (hunk 2 above)
Comment (jcr7467) [reply client/game.py:316]: if you check 2 lines below that I need the enemy type to get its health

File: client/game.py (hunk 1 above)
Comment (jcr7467) [reply client/game.py:525]: will do, didn't see that there

Comment (jcr7467) [reply client/game.py]: done
